docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,82 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in Kubernetes. 
+
+```shell
+kubectl delete -f <file.yaml> 
+```
+
+or
+
+```shell
+kubectl delete statefulsets <statefulset-name>
+```
+
+The associated headless service may need to be deleted separately after the Stateful Set itself is deleted.
+
+```shell
+kubectl delete service <service-name>
+```
+
+Deleting a Stateful Set through kubectl will scale it down to 0, thereby deleting all pods that are a part of it. If you wish to delete just the Stateful Set and not the pods, use `--cascade=false`.
+
+```shell
+kubectl delete -f <file.yaml> --cascade=false
+```
+Setting `--cascade=false` causes the Pods managed by the Stateful Set to be left behind even after the Stateful Set object itself is deleted.
+If the pods have a label `app=myapp`, they can then be deleted as follows.
```

Comment:
Passive voice. "They can then be deleted" -> "You can then delete them as follows:"